bundles/org.openhab.binding.souliss/README.md

```diff
@@ -0,0 +1,352 @@
+For updates go to official repository at [https://github.com/souliss/bindingopenhab2](https://github.com/souliss/bindingopenhab2)
+
+# Souliss Binding
+
+[Souliss](http://www.souliss.net/) is a networking framework for Arduino and compatibles boards, and is designed to let you easily build a smart home that is distributed over multiple boards via Ethernet, WiFi, wireless point-to-point and RS485 bus. 
+
+Souliss is an open-source and community driven project, you can use the [wiki](https://github.com/souliss/souliss/wiki) and [Community](https://github.com/souliss/souliss/wiki/Community) to get help and share you results.  
+
+## Prerequisites
+
+The binding requires a deployed network.  As a minimum, you need one Souliss node with Ethernet or WiFi access configured as a [Gateway](https://github.com/souliss/souliss/wiki/Gateway). A Gateway is a special node that is able to communicate with the user interfaces. The binding interacts as a user interface for Souliss.
```

Comment:
Please put all sentence's on their own line.
This helps when lines change in the future.
Do this for all of the readme.

Reply:
Can you give me an example ? Not sure to understand it... One sentence for line ?